parlai/tasks/squad2/agents.py

```diff
@@ -23,6 +33,10 @@ class IndexTeacher(FixedDialogTeacher):
     location of the answer in the context.
     """
 
+    @classmethod
+    def add_cmdline_args(cls, argparser):
+        add_common_cmdline_args(argparser)
```

Comment:
Hmm are there params from this that we need to use in this teacher?

Reply:
yes --impossible-answer-string

Reply:
ha, makes sense :)